dev-perl/PGObject-Type-DateTime/PGObject-Type-DateTime-2.1.0.ebuild

```diff
@@ -0,0 +1,20 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DIST_AUTHOR="EINHVERFR"
+
+inherit perl-module
+
+DESCRIPTION="DateTime Wrappers for PGObject"
+
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="dev-perl/DateTime-TimeZone
+	dev-perl/DateTime
+	>=dev-perl/PGObject-2.0.1"
+
+BDEPEND="${RDEPEND}"
```

Comment:
This actually isn't proper. BDEPEND is for host dependencies (think ninja, cmake, yacc ...).
Since the package has no DEPEND, it can also happen that the order of packages comes out wrong (in this case build is a NO-OP, but that's not a given.

Also, I think in this package we don't need to set BDEPEND at all as the eclass `perl-module` adds perl-Test-Simpe and dev-lang/perl.
Either way, Im going to fix and merge.